centreon/tests/e2e/features/Dashboards/14-resource-table-widget-configuration/index.ts

```diff
@@ -532,3 +541,77 @@ Then(
     cy.contains('host2').should('exist');
   }
 );
+
+Given('a dashboard containing a resource table widget', () => {
+  cy.logoutViaAPI()
+  cy.loginByTypeOfUser({
+    jsonName: 'admin',
+    loginViaApi: false
+  });
+  cy.insertDashboardWithWidget(dashboards.default, resourceTable);
+  cy.editDashboard(dashboards.default.name);
+  cy.wait('@resourceRequest');
+  cy.editWidget(1);
+  cy.wait('@resourceRequest');
+});
+
+When('the dashboard administrator clicks on a random resource from the resource table', () => {
+  cy.get('[aria-label^="Select row"]').eq(0).click()
```

Comment:
use getbylabel function

Reply:
Using [aria-label^="Select row"] selects elements where the aria-label attribute begins with the string 'Select row'. However, this approach isn't compatible with the getByLabel function for such selections.